src/test/java/com/liorkn/elasticsearch/PluginTest.java

```diff
@@ -71,8 +71,8 @@ public void test() throws Exception {
         final Map<String, String> params = new HashMap<>();
         params.put("refresh", "true");
         final ObjectMapper mapper = new ObjectMapper();
```

Comment:
please update this file with master - it's missing some assertions at the end

Reply:
It should be the same now